docs/lib/datastore/getting-started.md

```diff
@@ -9,44 +9,155 @@ Amplify DataStore provides a programming model for leveraging shared and distrib
 
 <amplify-callout>
 
-DataStore can be used as a local-only data persistence mechanism and an AWS account is not required for that. However, if you wish to sync with the cloud it is recommended you [Install and configure the Amplify CLI](~/cli/start/install.md).
+**Note:** cloud synchronization is optional and disabled by default. This allows you to start with DataStore right away, even without an AWS account.
 
 </amplify-callout>
 
-## Generate models
+## Goal
+To setup and configure your application with Amplify DataStore and use it to persist data locally on a device.
 
-Modeling your data and *generating models* which are used by DataStore is the first step to get started. GraphQL is used as a common language across all supported platforms for this process, and is also used as the network protocol when syncing with the cloud. GraphQL is also what powers some of the features such as [Automerge in AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/conflict-detection-and-sync.html#automerge).
+## Prerequisites
+
+<inline-fragment platform="js" src="~/lib/datastore/fragments/js/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/10_preReq.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/10_preReq.md"></inline-fragment>
+
+## Install Amplify Libraries
+
+<inline-fragment platform="ios" src="~/lib/datastore/fragments/ios/getting-started/20_installLib.md"></inline-fragment>
+<inline-fragment platform="android" src="~/lib/datastore/fragments/android/getting-started/20_installLib.md"></inline-fragment>
+
+There are two options to integrate the Amplify build process with the project.
+
```

Comment:
I think there is a potential problem here.  In our pre-req, we have them go through a project setup, which has them manually adding amplifyconfiguration.json and awsconfiguration.json into the project.  This seems to conflict with the instructions here...

option 1:
* put this option1/option2 thing into the project setup
* then assume they have done one or the other here, and update the verbage

Option 2:
* do not reference amplify tools in this guide.
* leave project setup as-is

Option 3:
* hmmm something else?

Reply:
Fixing this some other time, resolving.